README.md

```diff
@@ -100,12 +100,44 @@ spec:                  # spec (used to be che-plugin.yaml)
       env:               # list of env vars to set in sidecar
         - name:
           value:
+      command:           # optinal; definition of root process command inside container
+        - /bin/sh
+      args:              # optinal; list arguments for root process command inside container
+        - -c
+          ./entrypoint.sh
       volumes:           # volumes required by plugin
         - mountPath:
           name:
+          persistVolume: # boolean; if true volume will be persisted, otherwise volume will be ephemeral; default value is true
```

Comment:
Following the discussion [here](https://github.com/eclipse/che-plugin-broker/pull/74#discussion_r326644727), I would go with a more k8s-like name here with a default value of false. I suggested `emptyDir` but `ephemeral` also works.